src/integrationTest/groovy/wooga/gradle/unity/tasks/GenerateSolutionTaskIntegrationSpec.groovy

```diff
@@ -0,0 +1,41 @@
+package wooga.gradle.unity.tasks
+
+import com.wooga.spock.extensions.unity.UnityPathResolution
+import com.wooga.spock.extensions.unity.UnityPluginTestOptions
+import com.wooga.spock.extensions.uvm.UnityInstallation
+import net.wooga.uvm.Installation
+import spock.lang.Requires
+import wooga.gradle.unity.UnityTaskIntegrationSpec
+
+class GenerateSolutionTaskIntegrationSpec extends UnityTaskIntegrationSpec<GenerateSolution> {
+
+    @UnityPluginTestOptions(addMockTask = false, disableAutoActivateAndLicense = false)
+    def "calls unity generateSolution method from command line"() {

```

Comment:
I think the description could be clearer, as this test is not  about invoking generateSolution method, but about being able to execute the GenerateSolution task and that it sets the right arguments. 

(Since the output is checked in the other test)

Reply:
I also don't think the task is necessarily invoked from the CLI. We could use a GUI to have them executed!

Reply:
It isnt the task, it is the unity method itself that is being called from the CLI.

Reply:
I think both statements are correct; I think either one can be in the description.